### PR TITLE
Queue Consumption On Load

### PR DESCRIPTION
**Pull request by Anurag-Raut**
This PR Fixes issue #32 

As discussed with @DanielleMiu, initialized Blockinterval Directly rather than setting it null.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ document.addEventListener("blue-blocker-event", function (e) {
 	// retrieve option
 	api.storage.sync.get(DefaultOptions, items => {
 		SetOptions(items);
+
```
Comment (DanielleMiu) [inline script.js:9]: this change isn't necessary

**Review by DanielleMiu — CHANGES_REQUESTED** [review]
could you add your testing details?
I usually add the following to the bottom of the pr
```
- [ ] tested on chrome
- [ ] tested on firefox
```
in this case I might mention something about how when the page started the queue started being completed immediately

**Comment by Anurag-Raut** [comment]
> 

How can I prove that it is working?

**Comment by Anurag-Raut** [comment]
> > 
> 
> How can I prove that it is working? I was trying by loading page so that block items are pushed in blockqueue and then quickly returning back before they are are poped out , and then again going back to check if the items already present in queue are popped out and blocked.

**Comment by DanielleMiu** [comment]
that's basically what I'm looking for, like
> populated queue, closed tab, opened new twitter tab, no items were added to queue but existing queue was processed
would be totally sufficient. testing on both platforms would be best, but I do that before I push a new version regardless so not a dealbreaker.

**Pull request closed without merging**